kale/predict/decode.py

```diff
@@ -32,3 +43,122 @@ def forward(self, x):
         x = F.relu(self.fc3(x))
         x = self.fc4(x)
         return x
+
+
+class MultiRelaInnerProductDecoder(torch.nn.Module):
+    """
+    Build `DistMult
+    <https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/ICLR2015_updated.pdf>`_ factorization as GripNet decoder in PoSE dataset.
```

Comment:
Does it have a DOI?

Reply:
Shall we consider merging `decode.py` and `factorization.py`? https://github.com/pykale/pykale/blob/main/kale/embed/factorization.py

Reply:
Note one is under `predict` and the other is under `embed`. Explore in a discussion and/or a separate PR.

Reply:
@NYXFLOWER If this is an ICLR paper, there should be a DOI to use. Please check and let me know why that link is better than DOI.

Reply:
I can not find the DOI of this paper online. ICLR 2015 papers are published on arXiv only. See the official web here: https://iclr.cc/archive/www/2015.html.